.github/workflows/auto-release.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  pull_request:
+name: auto-release
+jobs:
+  approve:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/github-script@v3.0.0
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          // only approve PRs from release-please
+          if (context.payload.sender.login != "release-please") {
+            return;
+          }
+          // only approve PRs like "chore: release <release version>"
+          if (!context.payload.pull_request.title.startsWith("chore: release")) {
+            return;
+          }
+          // trigger auto-release when
+          // 1) shared-dependencies bom is going out;
+          // 2) there are no open dependency update PRs in this repo (to avoid multiple releases)
+          if (context.payload.pull_request.body.contains("com.google.cloud:google-cloud-shared-dependencies") && (
```

Comment:
Should we generalize this to be:

If it's a dependency only release and there's no open dependency bump pull requests

rather than specifically for the shared-dependencies version?

Reply:
done